src/internal/modules/billing/controller.js

```diff
@@ -1,13 +1,15 @@
 const uuid = require('uuid/v4');
 const { selectBillingTypeForm, billingTypeFormSchema } = require('./forms/billing-type');
 const { selectBillingRegionForm, billingRegionFormSchema } = require('./forms/billing-region');
-const { viewBillRunListForm } = require('./forms/bill-run-list');
+const { batchListForm } = require('./forms/billing-batch-list');
 const services = require('internal/lib/connectors/services');
 const forms = require('shared/lib/forms');
 const { get } = require('lodash');
 const moment = require('moment');
 const queryString = require('querystring');
 const helpers = require('@envage/water-abstraction-helpers');
+const Regions = require('./lib/regions');
```

Comment:
Just wondering if Regions here is a class - generally we would use camelcase if not

Reply:
camel case applied